vars/centos-8.yml

```diff
@@ -14,5 +14,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+_rpm_repository:
```

Comment:
`_rpm_repository` is used for initializing yum repo with upstream provided `repo.rpm`, not for installing the dependency nor package itself :-(